heltour/tournament/admin.py

```diff
@@ -661,7 +661,9 @@ def player_info_view(self, request, object_id, player_name):
             'league': season.league,
             'player': season_player.player,
             'reg': reg,
-            'has_played_20_games': has_played_20_games
+            'has_played_20_games': has_played_20_games,
+            'date_created': sp.registration.friends if sp.registration else None,
```

Comment:
`sp` is not defined here, you'll either need to expand it to `season_player` or rename the variable above